profiles/disa_stig-rhel7-baseline/controls/V-71859.rb

```diff
@@ -110,8 +110,9 @@
   [org/gnome/login-screen]
   banner-message-enable=true"
 
-  describe command("grep banner-message-enable /etc/dconf/db/local.d/*") do
-    its('stdout') { should cmp 'banner-message-enable=true'}
+  only_if { command('dconf').exist? }
+
+  describe command("dconf read /org/gnome/login-screen/banner-mesage-enable") do
```

Comment:
I just checked and, sure enough, the user gets a 'permission denied' error if they try to override something that has been locked.